bin/offsetting/Cargo.toml

```diff
@@ -18,3 +19,5 @@ x-flipper-360 = { path = "../../lib/x-flipper-360" }
 soiboy = { path = "../../lib/soiboy" }
 matryoshka = { path = "../../lib/matryoshka" }
 indctive = { path = "../../lib/indctive" }
+whynow = { path = "../../lib/whynow" }
+log = "0.4.26"
```

Comment:
We currently don't hava a logging implementation and just uesed `print`. https://github.com/offsetting/offsetting/blob/main/bin/offsetting/src/x360.rs#L78